File: BirdWatcherWeb/Models/WatcherHealthCheck.cs
using System;

namespace BirdWatcherWeb.Models
{
    public class WatcherHealthCheck
    {
        public long Id { get; set; }
        public string Hostname { get; set; }
        public string IP { get; set; }
        public float CPU_Temp { get; set; }
        public float GPU_Temp { get; set; }
        public long Disk_Total { get; set; }
        public long Disk_Used { get; set; }
        public long Disk_Free { get; set; }
        public DateTime Timestamp { get; set; }
    }
}

File: WatcherHealthCheck/watcherhealthcheck.py
class WatcherHealthCheck(object):
    def __init__(self, Hostname, IP, CPU_Temp, GPU_Temp, Disk_Total, Disk_Used, Disk_Free, Timestamp):
        self.Hostname = Hostname
        self.IP = IP
        self.CPU_Temp = CPU_Temp
        self.GPU_Temp = GPU_Temp
        self.Disk_Total = Disk_Total
        self.Disk_Used = Disk_Used
        self.Disk_Free = Disk_Free
        self.Timestamp = Timestamp